README.md

```diff
@@ -0,0 +1,31 @@
+# prplMesh Agent
+The prplMesh Agent implements a Multi-AP v1.0 certifiable agent (work in progress).
+
+# Build
+Build system is CMAKE, and supports RDKB and UGW cross compilation builds.
+Building natively (Linux only) is partially supported only for development process. Some of the components are skipped / built as stubs in this mode.
+
+# Run agent on Linux
+Running the agent on Linux allows easier development process without the need for real hardware. Instead, mac80211_hwsim can be used to simulate WLAN radios. A dummy bridge named br-lan must also exist since the agent uses the bridge MAC as the MultiAP AL-MAC.
+Last, the controller needs to run either on the same host or in another host with L2 connectivity (provided the BTL is using the local bus and not UDS mode).
+
+Currently, this mode was tested on an Ubuntu18.04 with hwsim, the agent was compiled to UDS mode.
+
+## Prerequisites
+- Interface exists for every slave in the configuration file
+- bridge named br-lan (can be empty)
+- agent, controller, common and framework built natively (see prplMesh-tools for build instructions)
+
+One time setup `modprobe mac80211_hwsim; brctl addbr br-lan`
+
+Build all components `cd tools; python3 ./maptools.py build all`
```

Comment:
No MSGLIB needed?

Reply:
No, it is not needed since we run controller and agent on the same box. We can build with message library but it is not mandatory unless we are testing 2 different systems, one running a controller (+agent optionally), the other running agent only.
For the sake of simplicity, and since currently we do not support building nng as part of maptools, better stick to UDS implementation.

Later on once everything works the same with nng which is built as part of maptools, we can update the README.